### PR TITLE
Add dashboard view

### PR DESCRIPTION
**Pull request by ktkization**

### CODE REVIEW

File: webapp/lib/view.dart [hunk 1]
```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
...
+          ['Needs: Q/A, Labelling team, Safeguarding response', 'Produces: Dashboard for distribution of answers']),
+        new AvailablePackagesViewPartial('Information Service',
+          'Answer people\'s questions',
+          ['Needs: Response protocol, Labelling team, Safeguarding response', 'Produces: Thematic distribution, work rate tracker']),
+        new AvailablePackagesViewPartial('Bulk Message',
+          'Send set of people a once off message',
+          ['Needs: Definition of who. Safeguarding response', 'Produces: Success/Fail tracker'])
+      ]);
+
+      dashboardView.pupolateActivePackages();
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: typo nit: should be `populateActivePackages`

File: webapp/lib/view.dart [hunk 2]
```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
...
   ContentView() {
-    contentElement = new DivElement();
+    contentElement = new DivElement()..classes.add('content');
+    dashboardView = new DashboardView();
+    dashboardView.activePackages.addAll(
+      [
+        new ActivePackagesViewPartial('Urgent conversations'),
+        new ActivePackagesViewPartial('Open conversations'),
+        new ActivePackagesViewPartial('Batch replies (Week 12)'),
+      ]);
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart:55]: these should be generated in the controller not here, as we likely want them to be stored in and read from a config firebase collection - something for a next PR maybe?

File: webapp/lib/view.dart [hunk 3]
```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
...
+      ..classes.add('available-packages');
+
+    _projectActionsContainer.append(_conversationsLink);
+    _projectActionsContainer.append(_oversightDashboardLink);
+    dashboardViewElement.append(_projectActionsContainer);
+    dashboardViewElement.append(activePacakgesContainer);
+    dashboardViewElement.append(availablePackagesContainer);
+  }
+
+  void pupolateActivePackages() {
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: consider renaming to `renderActivePackages`

File: webapp/lib/view.dart [hunk 4]
```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
...
+    activePacakgesContainer.children.clear();
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart]: typo nit: should be `activePackagesContainer`

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization , LGTM with a couple of nits and a suggestion for a future PR

File: webapp/lib/view.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Oops. Sorry for the spelling mistakes. Renamed it to render as per your other suggestion.

File: webapp/lib/view.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/view.dart:55]: Yes. I can do this in the next PR. Just wanted some placeholders for the UI.

File: webapp/lib/view.dart (hunk 3 above)
Comment (ktkization) [reply webapp/lib/view.dart]: That better. Renamed also `populateAvailablePackages` to `renderAvailablePackages`

File: webapp/lib/view.dart (hunk 4 above)
Comment (ktkization) [reply webapp/lib/view.dart]: Oops. Sorry for the spelling mistakes. Corrected.

Comment (marianamarasoiu) [reply webapp/lib/view.dart]: no worries, it happens!

File: webapp/lib/view.dart (hunk 2 above)
Comment (marianamarasoiu) [reply webapp/lib/view.dart:55]: cool, thanks!

**Review by marianamarasoiu — APPROVED** [review]
thanks @ktkization ! approving and merging it now

**Pull request merged**